group4blandine/group4

```diff
@@ -0,0 +1,7 @@
+Sinclair
```

Comment:
Sinclair I did understood why we did mkdir instead of doing git checkout directly?

Reply:
Because Eric don't want us to mess up the repo.

We can mess up in our own directory.

Reply:
Oh ok I understand now